modules/distr-tracing-tempo-config-multitenancy.adoc

```diff
@@ -159,9 +159,21 @@ spec:
             authenticator: bearertokenauth
           headers:
             X-Scope-OrgID: "dev"
+        otlphttp/dev: # <2>
+          endpoint: https://tempo-simplest-gateway.chainsaw-multitenancy.svc.cluster.local:8080/api/traces/v1/dev
+          tls:
+            insecure: false
+            ca_file: "/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt"
+          auth:
+            authenticator: bearertokenauth
+          headers:
+            X-Scope-OrgID: "dev"
       service:
         extensions: [bearertokenauth]
         pipelines:
           traces:
-            exporters: [otlp/dev]
+            exporters: [otlp/dev] # <3>
```

Comment:
The brackets are needed for a real value?

Reply:
Thank you for double-checking. This isn't a placeholder, it's the real value.